src/state/schema.ts

```diff
@@ -22,6 +25,7 @@ export const MusicCollectionSchema = z.object({
 	version: z.literal(MusicCollectionVersion),
 	title: z.string(),
 	items: z.array(MusicItemSchema),
+	owner: DidSchema,
```

Comment:
Starting to track the owners of the collections as it will be the building block for sharing.

Not sure we will be able to read this value easily, we may need some metadata storage.